### ISSUE TITLE
Show other users cursors on host pc

### ISSUE THREAD

**Issue by Bommel24** [rustdesk/rustdesk#573]
Would be a nice feature to see other users cursors like in AnyDesk maybe in different colors and the username written below.

**Comment by redpenguinyt** [comment]
in addition it would be helpful for the cursor to blink in some way to indicate when the other user clicks, like in anydesk

**Comment by davidnmbond** [comment]
Would definitely like this option!

**Comment by danielabbatt** [comment]
Yes please. this would make it so much easier than trying to describe to someone where on the screen you want them to do something.

**Comment by rustdesk** [comment]
This is a good example of how to get your feature request higher priority.

https://twitter.com/rustdesk/status/1770643072118095914

**Comment by FloMaetschke** [comment]
still not working in end of 2024. will stick with teamviewer.

**Comment by rustdesk** [comment]
> if there’s a way to show a remote pointer or highlight the mouse position while being in view-only mode. My goal is to guide the user by pointing to elements on their screen, without controlling their mouse or keyboard. This is especially useful for training or support situations where user interaction is required, but full control is not desirable.
